client/src/App.js

```diff
@@ -1,12 +1,230 @@
-import React, {useEffect, useState} from 'react';
-import logo from './logo.svg';
-import Ticket from './components/Ticket'
+import React, { useState, useEffect } from 'react';
+import InputLabel from '@material-ui/core/InputLabel';
+import { TextField } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import NativeSelect from '@material-ui/core/NativeSelect';
+import axios from 'axios';
+import Ticket from './components/Ticket';
+import Header from './components/Header';
 import './App.css';
 
 function App() {
+  const [TicketArr, setTicketArr] = useState([]);
+  const [sortByTime, setSortByTime] = useState([[], 'default']);
+  const [dateRange, setDateRange] = useState(
+    {
+      from: (new Date(2017, 11, 1)).getTime(),
+      to: Date.now(),
+    },
+  );
+  const [showFinished, setShowFinished] = useState(false);
+  // const ActiveTickets = sortByTime[0].filter((ticket) => !ticket.done && !ticket.hidden);
+  // const HandeledTickets = sortByTime[0].filter((ticket) => ticket.done && !ticket.hidden);
+  let ActiveTickets =[], HandeledTickets = [],hiddenCounter1=0 , hiddenCounter2 =0;
+  sortByTime[0].forEach(ticket => {
+    if(ticket.hidden && !ticket.done){
+      hiddenCounter1++;
+    }else if(ticket.hidden && ticket.done){
+      hiddenCounter2++;
+    }
+      if(ticket.done){
+        HandeledTickets.push(ticket);
+      }else{
+        ActiveTickets.push(ticket);
+      }
+  })
+  //const hiddenCounter1 = sortByTime[0].length - (ActiveTickets.length + HandeledTickets.length);
+
+  function oldOrNew(val, arr) {
+    const arrCopy = arr ? arr.slice() : TicketArr.slice();
+    const newArr = arrCopy.filter((ticket) => ticket.creationTime < dateRange.to && ticket.creationTime > dateRange.from);
+    if (val === '0') {
```

Comment:
What is this `val` variable? I cant understand from the name or from the value.
Why is its value is a stringed number? what does it represents? cant it be represented better in other format?